File: src/luogu/p1125/Main.java
package luogu.p1125;

import java.util.HashMap;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int maxn = Integer.MIN_VALUE;
        int minn = Integer.MAX_VALUE;
        Scanner sc = new Scanner(System.in);
        String wd = sc.next();
        HashMap<Character, Integer> map = new HashMap<>();
        for (char c : wd.toCharArray()) {
            if (!map.containsKey(c)) {
                map.put(c, 0);
            }
            map.put(c, map.get(c) + 1);
        }
        for (Integer value : map.values()) {
            if (value > maxn)
                maxn = value;
            if (value < minn)
                minn = value;
        }
        if (isPrime(maxn - minn)) {
            System.out.println("Lucky Word");
            System.out.println(maxn - minn);
        } else {
            System.out.println("No Answer");
            System.out.println(0);
        }

    }

    public static boolean isPrime(int i) {
        if (i < 2)
            return false;
        for (int j = 2; j < i / 2 + 1; j++) {
            if (i % j == 0) {
                return false;
            }
        }
        return true;
    }
}

File: src/luogu/p1125/Main.cpp
// https://www.luogu.com.cn/problem/P1125
#include <cmath>
#include <iostream>
#include <map>
using namespace std;
bool isPrime(int i) {
  if (i < 2) return false;
  for (int j = 2; j < i / 2 + 1; j++) {
    if (i % j == 0) {
      return false;
    }
  }
  return true;
}
int main() {
  string s;
  cin >> s;
  map<char, int> m;
  int maxn = -1;
  int minn = 120;
  for (int i = 0; i < s.length(); ++i) {
    m[s[i]]++;
  }
  for (auto &&i : m) {
    maxn = max(maxn, i.second);
    minn = min(minn, i.second);
  }
  if (isPrime(maxn - minn)) {
    cout << "Lucky Word" << endl;
    cout << maxn - minn << endl;
  } else {
    cout << "No Answer" << endl;
    cout << 0 << endl;
  }
  return 0;
}